docs/_posts/2021-02-13-hudi-key-generators.md

```diff
@@ -5,18 +5,16 @@ author: shivnarayan
 category: blog
 ---
 
-Every record in Hudi is uniquely identified by a HoodieKey, which is a pair of record key and partition path where the 
-record belongs to. Hudi has imposed this constraint so that updates and deletes can be applied to the record of interest. 
-Hudi relies on the partition path field to partition your dataset and records within a partition have unique record keys. 
-Since uniqueness is guaranteed only within the partition, there could be records with same record keys across different 
-partitions. One should choose the partition field wisely as it could be a determining factor for your ingestion and 
-query latency.
+Every record in Hudi is uniquely identified by a primary key, which is a pair of record key and partition path where
+the record belongs to. Using primary keys, Hudi can impose a) partition level uniqueness integrity constraint
```

Comment:
should we mention about global index here which can help achieve uniqueness across partitions?

Reply:
yeah. I did have it in my initial draft. But guess nishith feedback was to not go into too much details. let me sync up w/ him again.